src/Command/GenerateAdminCommand.php

```diff
@@ -1,334 +0,0 @@
-<?php
-
-declare(strict_types=1);
```

Comment:
We should not remove classes, because this is not BC.

Reply:
Are you sure? The class is unusable with Symfony 4

Reply:
If you extend it and make some modifications, it could work with symfony

Reply:
You would have to override and stop calling `parent::isEnabled()` and `parent::execute()`. That does not sound realistic.

Reply:
In this case the should migrate to maker command and extend here as it’s the new best practice and some kind of standard, instead of putting the effort in overwriting/hacking some methods.

We can ofc mention this in the upgrade file

Reply:
I added something to the upgrade file, please have a look.